src/collection/pages/main/main.tsx

```diff
@@ -248,7 +248,7 @@ const Main: React.FC = () => {
       (x) => x.collectionPhase === CollectionPhase.LATEST
     );
 
-    if(!latestCollectionItem) return;
+    if (!latestCollectionItem) throw Error("Mutation target cannot be 0");
```

Comment:
is it intented?

Reply:
It isn't intended. 😢 I'll open another PR to address it.